assets/sass/components/adminbar/_googlesitekit-adminbar.scss

```diff
@@ -65,16 +90,6 @@
 		}
 	}
 
-	// Display sub menu on hover.
-	.googlesitekit-wp-adminbar.hover .googlesitekit-adminbar,
-	&.nojs .googlesitekit-wp-adminbar:hover .googlesitekit-adminbar {
-		display: block;
-
-		&.--has-error {
-			display: none;
-		}
-	}
-
```

Comment:
@Shavindra I removed these because they were unnecessarily reimplementing hide/show logic which is already handled by core using the `ab-sub-wrapper` class which was already present but not in the right place. This made the previous `&:not(.nojs) ...` styles redundant as well.

Reply:
Ah right ok!